doc/source/tokudb/tokudb_background_analyze_table.rst

```diff
@@ -0,0 +1,146 @@
+.. _tokudb_background_analyze_table:
+
+===============================
+TokuDB Background ANALYZE TABLE
+===============================
+
+Prior to :rn:`5.6.27-76.0` release there was no mechanism to optionally and automatically trigger a background analysis or gathering of cardinality statistics on a |TokuDB| tables. These statistics are used for ``JOIN`` optimizations and helping the |MySQL| optimizer choose the appropriate index for a query. If a table's statistics or index cardinality becomes outdated, you might see queries which previously performed well suddenly show up on slow query log until the statics are updated again. For |TokuDB| cardinality statistics were only updated when an explicit ``ANALYZE TABLE`` was issued.
```

Comment:
Queries appearing on slow query log is a symptom of the issue (slow query), not an issue by itself.